examples/misc/lib/language_tour/classes/orchestra.dart

```diff
@@ -38,6 +38,7 @@ abstract class Performer {
 // #docregion Musician-and-Maestro
 class Musician extends Performer with Musical {
   // #enddocregion Musician-and-Maestro
+//  Musician() : super('Anonymous');
```

Comment:
Was the intent to leave this comment here?

Reply:
Ha, no. :)